File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListBox/Appearance/TextWithColorPage.vb
Namespace CompanionKit.Controls.ListBox.Appearance

	Public Class TextWithColorPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

			'Set collection of customer as DataSource.
            Me.mobjListBox.DataSource = Global.CompanionKit.Controls.Utils.CustomerData.GetCustomers

            'Fill TextBoxes with the values of appropriate properties
            Me.mobjDisplayTextBox.Text = Me.mobjListBox.DisplayMember
            Me.mobjValueTextBox.Text = Me.mobjListBox.ValueMember
            Me.mobjColorTextBox.Text = Me.mobjListBox.ColorMember
		End Sub

    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListBox/Appearance/TextWithColorPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.ListBox.Appearance
{
    public partial class TextWithColorPage : UserControl
    {
        public TextWithColorPage()
        {
            InitializeComponent();

            //Set collection of customer as DataSource.

            this.mobjListBox.DataSource = Gizmox.WebGUI.Forms.CompanionKit.Controls.Util.CustomerData.GetCustomers();

            //Fill TextBoxes with the values of appropriate properties
            this.mobjDisplayTextBox.Text = this.mobjListBox.DisplayMember;
            this.mobjValueTextBox.Text = this.mobjListBox.ValueMember;
            this.mobjColorTextBox.Text = this.mobjListBox.ColorMember;
        }
    }
}